pkg/logging/logger.go

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) The Thanos Authors.
+// Licensed under the Apache License 2.0.
+
+package logging
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+)
+
+const (
+	LogFormatLogfmt = "logfmt"
```

Comment:
```suggestion
	FormatLogfmt = "logfmt"
```
We can use package name now :hugs:

Reply:
hmm not sure, I actually think it's clearer with the prefix